cms/djangoapps/contentstore/rest_api/v1/views/home.py

```diff
@@ -80,7 +81,12 @@ def get(self, request: Request):
 
         home_context = get_home_context(request, True)
         home_context.update({
-            'allow_to_create_new_org': settings.FEATURES.get('ENABLE_CREATOR_GROUP', True) and request.user.is_staff,
+            # 'allow_to_create_new_org' is actually about auto-creating organizations
+            # (e.g. when creating a course or library), so we add an additional test.
+            'allow_to_create_new_org': (
+                home_context['can_create_organizations'] and
+                org_api.is_autocreate_enabled()
+            ),
```

Comment:
For reference, the `can_create_organizations` value is set here:

https://github.com/openedx/edx-platform/blob/c5fabd586376478b8c5b784c3bd7ce5264c259dd/cms/djangoapps/contentstore/views/course.py#L1794-L1798

Reply:
@pomegranited This seems reasonable but the logic while creating courses seems different. If the setting to create new org is set to `False`, the API only allows user/staff to create courses in organizations they are allowed in: 
* https://github.com/open-craft/frontend-app-course-authoring/blob/a25a09e053ea0e0d153bfd28280072ebe6e5475f/src/generic/create-or-rerun-course/hooks.jsx#L35
* https://github.com/open-craft/edx-platform/blob/1ac6e02bb8a16ac76bc0f2e4d25626e4e0734c01/cms/djangoapps/contentstore/utils.py#L1582

We also have [get_allowed_organizations_for_libraries](https://github.com/open-craft/edx-platform/blob/1ac6e02bb8a16ac76bc0f2e4d25626e4e0734c01/cms/djangoapps/contentstore/utils.py#L1581) for libraries, so probably we need to do the same here.

Reply:
Good catch @navinkarkera !  see [f1f40e0](https://github.com/openedx/edx-platform/pull/36094/commits/f1f40e0e87fdd37aa147770fa3b197175832c50f)